pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: boolean_selector
-version: 1.0.3-dev
+version: 1.0.4-dev
```

Comment:
Don't think this is needed, since `1.0.3` has not been released

Reply:
Yeah, this should just be `1.0.3`.